src/main/resources/builtin/materialValues.yml

```diff
@@ -4,21 +4,28 @@ default:
   lightOpacity: 255
   flameResistance: -1
   fireResistance: -1
+  baseMapColor: 44
 values:
+  # TODO: Define baseMapColor for all materials
```

Comment:
The material values file is a WIP for all properties, so I feel like this comment is confusing.